card/mod/04_settings/set/self/follow_fields.rb

```diff
@@ -0,0 +1,2 @@
+extend Card::Setting
+setting_opts :group=> :config, :position=>5, :rule_type_editable=>false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Final newline missing.